content/ember/v3/deprecate-get-with-default.md

```diff
@@ -0,0 +1,78 @@
+---
+id: ember-metal.get-with-default
+title: Use Ember getter and explicitly check for undefined
+until: '4.0.0'
+since: '3.19'
+---
+
+Deprecate support for `getWithDefault` in Ember's Object module (@ember/object) – both the [function](https://api.emberjs.com/ember/release/functions/@ember%2Fobject/getWithDefault) and the [class method](https://api.emberjs.com/ember/release/classes/EmberObject/methods/getWithDefault?anchor=getWithDefault) – because its expected behaviour is confusing to Ember developers.
+
+- The API will only return the default value when the value of the property retrieved is `undefined`. This behaviour is often overlooked when using the function where a developer might expect that `null` or other _falsy_ values will also return the default value.
```

Comment:
```suggestion
- The API will only return the default value when the value of the property retrieved is `undefined`. This behaviour is often overlooked when using the function where a developer might expect that `null` or other _falsey_ values will also return the default value.
```

I don't really know how to spell this non-word, but we use this spelling just below so we ought to at least stay consistent.

Reply:
:)